joplin/api/schema.py

```diff
@@ -877,6 +878,9 @@ def resolve_slug(self, resolve_info, *args, **kwargs):
     def resolve_page_type(self, info):
         return self.page.content_type.name
 
+    def resolve_live(self, info):
+        return get_page_from_content_type(self).live
```

Comment:
looked at `get_global_id_from_content_type` and it seems like it and `get_page_from_content_type` have some shared logic, might be cool to clean that stuff up at some point in the future (def not something that should happen before shipping this fix)